### PR TITLE
TW 261 Code snippets page refresh

### PR DESCRIPTION
**Pull request by HeatherStenson**

### CODE REVIEW

**Review by mark-anthony-postman — APPROVED** [review]
Looks great! 🦄

File: src/pages/docs/sending-requests/generate-code-snippets.md [hunk 1]
```diff
@@ -42,33 +42,34 @@ warning: false
 
 ---
 
-You can generate code snippets in various languages and frameworks within Postman.
+Postman can convert an API request into a code snippet in the programming language or framework of your choosing. You can use this generated code snippet in your front-end applications.
+
+> This guide contains information about generating code snippets from API requests. For information about generating server-side boilerplate code from an OpenAPI 3 specification. see [Generating server code](/docs/designing-and-developing-your-api/generating-server-code/).
```
Comment (jkonrath-postman) [inline src/pages/docs/sending-requests/generate-code-snippets.md:47]:
```suggestion
> This guide contains information about generating code snippets from API requests. For information about generating server-side boilerplate code from an API specification. see [Generating server code](/docs/designing-and-developing-your-api/generating-server-code/).
```

Comment (jkonrath-postman) [reply src/pages/docs/sending-requests/generate-code-snippets.md:47]: Make this generic so we don't have to hunt it down when OpenAPI 4 comes out someday.

**Review by jkonrath-postman — APPROVED** [review]
LGTM! (I am so glad that giant outdated list of languages is gone!)

File: src/pages/docs/sending-requests/generate-code-snippets.md [hunk 2]
```diff
@@ -42,33 +42,34 @@ warning: false
...
-
-<img src="https://assets.postman.com/postman-docs/context-menu-icon-v8.jpg" width="400px" alt="User Menu"/>
+1. With the request you want to use for a code snippet open, select the code icon <img alt="Code icon" src="https://assets.postman.com/postman-docs/icon-code-snippet.jpg#icon" width="16px"> in the right panel.
 
-Select a language to view and copy your generated code snippet.
+    <img src="https://assets.postman.com/postman-docs/generate-code-right-sidebar-v9.jpg" width="300px" alt="Open code snippets"/>
 
-![Code Language](https://assets.postman.com/postman-docs/code-language-v8.jpg)
+1. Use the dropdown list to select a [language or framework](#supported-languages-and-frameworks).
+1. Select the copy icon to copy the code snippet to your clipboard.
```
Comment (HeatherStenson) [inline src/pages/docs/sending-requests/generate-code-snippets.md:64]:
```suggestion
1. Select the copy icon <img alt="Copy icon" src="https://assets.postman.com/postman-docs/icon-copy-v9.jpg#icon" width="15px"> to copy the code snippet to your clipboard.
```

**Pull request merged**